rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -125,4 +125,30 @@ public static byte[] flatKeysAsByteArray(List<BtcECKey> keys) {
 
         return flatPubKeys;
     }
+
+    public static void addSignaturesToInput(BtcTransaction transaction, int inputIndex, List<BtcECKey> keys) {
+        if (transaction.getWitness(inputIndex).getPushCount() == 0) {
+            addSignaturesToInputScriptSig(transaction, inputIndex, keys);
+        }
+    }
+
+    private static void addSignaturesToInputScriptSig(BtcTransaction transaction, int inputIndex, List<BtcECKey> keys) {
```

Comment:
What if we name the method as `signInput...` instead of `addSignatures`?

Reply:
like it, thanks!

Reply:
updated